File: csharp/GardenCatalogue-TestDesign-Kata/GardenCatalogueTest/TestData.cs
using GardenCatalogue;

namespace GardenCatalogueTest;

public static class TestData
{
    public static IEnumerable<Plant> DefaultPlants => new List<Plant>
    {
        new PlantBuilder().WithName("Lavender").WithLatinName("Lavandula angustifolia").WithArticleNumber("FLO-001").WithBloomPeriod(BloomPeriod.FromRange(Month.June, Month.August)).WithMaxHeight(0.6).WithSoil(SoilCondition.Sandy).WithLight(LightCondition.FullSun).Build(),
        new PlantBuilder().WithName("Rose").WithLatinName("Rosa").WithArticleNumber("BSH-001").WithType(PlantType.Bush).WithBloomPeriod(BloomPeriod.FromRange(Month.June, Month.September)).WithMaxHeight(1.5).WithSoil(SoilCondition.Loamy).WithLight(LightCondition.FullSun).Build(),
        new PlantBuilder().WithName("Hydrangea").WithLatinName("Hydrangea macrophylla").WithArticleNumber("BSH-002").WithType(PlantType.Bush).WithBloomPeriod(BloomPeriod.FromRange(Month.July, Month.September)).WithMaxHeight(1.2).WithSoil(SoilCondition.Clay).WithLight(LightCondition.PartialShade).Build(),
        new PlantBuilder().WithName("Boxwood").WithLatinName("Buxus sempervirens").WithArticleNumber("BSH-003").WithType(PlantType.Bush).WithMaxHeight(1.0).WithSoil(SoilCondition.Any).WithLight(LightCondition.PartialShade).Build(),
        new PlantBuilder().WithName("Beech").WithLatinName("Fagus sylvatica").WithArticleNumber("TRE-001").WithType(PlantType.Tree).WithBloomPeriod(Month.May).WithMaxHeight(5.0).WithSoil(SoilCondition.Any).WithLight(LightCondition.FullSun).Build(),
        new PlantBuilder().WithName("Yew").WithLatinName("Taxus baccata").WithArticleNumber("TRE-002").WithType(PlantType.Tree).WithMaxHeight(4.0).WithSoil(SoilCondition.Any).WithLight(LightCondition.PartialShade).Build(),
        new PlantBuilder().WithName("Oak").WithLatinName("Quercus robur").WithArticleNumber("TRE-003").WithType(PlantType.Tree).WithMaxHeight(20.0).WithSoil(SoilCondition.Any).WithLight(LightCondition.FullSun).Build(),
        new PlantBuilder().WithName("Silver Birch").WithLatinName("Betula pendula").WithArticleNumber("TRE-004").WithType(PlantType.Tree).WithMaxHeight(15.0).WithSoil(SoilCondition.Sandy).WithLight(LightCondition.FullSun).Build(),
        new PlantBuilder().WithName("Fern").WithLatinName("Polypodiopsida").WithArticleNumber("BSH-004").WithType(PlantType.Bush).WithMaxHeight(0.5).WithSoil(SoilCondition.Loamy).WithLight(LightCondition.FullShade).Build()
    };
}

File: java/src/test/java/org/sammancoaching/TestData.java
package org.sammancoaching;

import java.util.List;
import java.util.Map;

public class TestData {
    public static List<Plant> defaultPlants() {
        return List.of(
            new PlantBuilder().withName("Lavender").withLatinName("Lavandula angustifolia").withArticleNumber("FLO-001").withBloomPeriod(BloomPeriod.fromRange(Month.June, Month.August)).withMaxHeight(0.6).withSoil(SoilCondition.Sandy).withLight(LightCondition.FullSun).build(),
            new PlantBuilder().withName("Rose").withLatinName("Rosa").withArticleNumber("BSH-001").withType(PlantType.Bush).withBloomPeriod(BloomPeriod.fromRange(Month.June, Month.September)).withMaxHeight(1.5).withSoil(SoilCondition.Loamy).withLight(LightCondition.FullSun).build(),
            new PlantBuilder().withName("Hydrangea").withLatinName("Hydrangea macrophylla").withArticleNumber("BSH-002").withType(PlantType.Bush).withBloomPeriod(BloomPeriod.fromRange(Month.July, Month.September)).withMaxHeight(1.2).withSoil(SoilCondition.Clay).withLight(LightCondition.PartialShade).build(),
            new PlantBuilder().withName("Boxwood").withLatinName("Buxus sempervirens").withArticleNumber("BSH-003").withType(PlantType.Bush).withMaxHeight(1.0).withSoil(SoilCondition.Any).withLight(LightCondition.PartialShade).build(),
            new PlantBuilder().withName("Beech").withLatinName("Fagus sylvatica").withArticleNumber("TRE-001").withType(PlantType.Tree).withBloomPeriod(new BloomPeriod(List.of(Month.May))).withMaxHeight(5.0).withSoil(SoilCondition.Any).withLight(LightCondition.FullSun).build(),
            new PlantBuilder().withName("Yew").withLatinName("Taxus baccata").withArticleNumber("TRE-002").withType(PlantType.Tree).withMaxHeight(4.0).withSoil(SoilCondition.Any).withLight(LightCondition.PartialShade).build(),
            new PlantBuilder().withName("Oak").withLatinName("Quercus robur").withArticleNumber("TRE-003").withType(PlantType.Tree).withMaxHeight(20.0).withSoil(SoilCondition.Any).withLight(LightCondition.FullSun).build(),
            new PlantBuilder().withName("Silver Birch").withLatinName("Betula pendula").withArticleNumber("TRE-004").withType(PlantType.Tree).withMaxHeight(15.0).withSoil(SoilCondition.Sandy).withLight(LightCondition.FullSun).build(),
            new PlantBuilder().withName("Fern").withLatinName("Polypodiopsida").withArticleNumber("BSH-004").withType(PlantType.Bush).withMaxHeight(0.5).withSoil(SoilCondition.Loamy).withLight(LightCondition.FullShade).build()
        );
    }
}
